### ISSUE TITLE
[Feature Request] Sort motor configurations alphabetically

### ISSUE THREAD

**Issue by OZ1SEJ** [openrocket/openrocket#2492]
**Is your feature request related to a problem? Please describe.**
When dealing with a lot of different motor configurations, it is unclear how the list is sorted, and clicking the column header ("Configuration") doesn't do anything;

**Describe the solution you'd like**
I would like to be able to sort the list alphabetically, either by default or by clicking the column header.

**Describe alternatives you've considered**
I believe the most viable alternative is to just not implement this feature.

**Additional context**
Screenshot attached here:

![Skærmbillede fra 2024-06-04 09-58-33](https://github.com/openrocket/openrocket/assets/2393164/0a70a90d-1c6e-45b1-9a1f-a2f54a3ac952)

**Comment by JoePfeiffer** [comment]
This would really be a nice feature

**Comment by neilweinstock** [comment]
Yes, this has bugged me for a long time.  We should be able to use the same sortable table as the sim results.
